worlds/dark_souls_3/__init__.py

```diff
@@ -175,7 +178,7 @@ def create_regions(self):
             dreg_heap_region.exits.append(Entrance(self.player, "Goto Ringed City", dreg_heap_region))
             self.multiworld.get_entrance("Goto Ringed City", self.player).connect(ringed_city_region)
 
-    # For each region, add the associated locations retrieved from the corresponding location_table
+    # For each region, add the associated locations retrieved from the corresponding =
```

Comment:
Is it a mistake or you wanted to write something else ?

Reply:
this was a typo, changed it back